experimental/c-cpp/README.md

```diff
@@ -0,0 +1,83 @@
+# Auto-build oss-fuzz projects from a GitHub repo URL
+
+
+## Overview
+
+The core part of approach in this module takes the following steps:
+
+1) An input in the form of a GitHub repo is given to `runner.py`.
+2) `runner.py` creates a temporary OSS-Fuzz project that contains
+  `build-generator.py`
+3) `runner.py` launches the OSS-Fuzz image of said project with
+  `build-generator.py` as entrypoint.
+
+`build-generator.py` will then proceed to carry out the following tasks within
+the Docker container:
```

Comment:
does this mean the OSS-Fuzz builder Docker container?

Reply:
Yup -- or more precisely it means the Docker image of a temporary project created that inherits from the base builder. This is the Dockerfile template we use for the temporary project: https://github.com/google/oss-fuzz-gen/pull/216/files#diff-86b63b45a6de7bc9bc127980ecfdc91f89646409b7d51ae960a9fe9d1730c97fR60-R70